src/AccordionItem/AccordionItem.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState, Fragment } from "react";
+import "./AccordionItem.scss";
+
+function AccordionItem({ title, body, id }) {
+  const [isActive, setIsActive] = useState(false);
+
+  function handleActivity() {
+    setIsActive(!isActive);
+  }
+
+  return (
+    <Fragment key={id}>
+      <div className="accordion">
+        <div className="accordion-title" onClick={handleActivity}>
+          {title}
+          <div>{isActive ? "▴" : "▼"}</div>
+        </div>
+        {isActive && <div className="accordion-content">{body}</div>}
```

Comment:
can't we use a semantic tag here? maybe p?

Reply:
Done.